scores_orchestration/orchestration.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import pkg_resources
+
+from spe_utils import utils
+
+from typing import Dict, Union, Optional
```

Comment:
Good use of typing variables.  Since typing in Python is optional and will work without it, do you use any special tools to ensure that you've specified types as often as possible?

Reply:
I try to use `mypy`.